src/components/tl/mlx5/tl_mlx5.c

```diff
@@ -77,6 +98,14 @@ static ucc_config_field_t ucc_tl_mlx5_context_config_table[] = {
      ucc_offsetof(ucc_tl_mlx5_context_config_t, devices),
      UCC_CONFIG_TYPE_STRING_ARRAY},
 
+    {"MCAST_TIMEOUT", "10000", "Timeout [usec] for the reliability NACK in Mcast",
+     ucc_offsetof(ucc_tl_mlx5_context_config_t, mcast_ctx_conf.timeout),
+     UCC_CONFIG_TYPE_INT},
+
+    {"MCAST_NET_DEVICES", "", "Specifies which network device to use for Mcast",
+     ucc_offsetof(ucc_tl_mlx5_context_config_t, mcast_ctx_conf.ib_dev_name),
+     UCC_CONFIG_TYPE_STRING},
```

Comment:
does it support single device or multiple devices? it's not clear since variable name is NET_DEVICES and description says "device".

Reply:
ok, if it's devices the config field type should be UCC_CONFIG_TYPE_STRING_ARRAY

Reply:
Actually we just support one device for now, I just double checked